Doc/library/pkgutil.rst

```diff
@@ -91,6 +91,10 @@ support.
    .. versionchanged:: 3.4
       Updated to be based on :pep:`451`
 
+   .. deprecated:: 3.12
```

Comment:
https://github.com/python/cpython/issues/97850 says "We'll deprecate them in 3.12 and remove them in 3.14", so:

```suggestion
   .. deprecated-removed:: 3.12 3.14
```